### ISSUE TITLE
Tokenizer - missing selection features

### ISSUE THREAD

**Issue by mikerodonnell89** [SAP/fundamental-ngx#2892]
Still missing a few features that UI5 has

- Users can select multiple tokens separately by holding down the CTRL key and selecting the relevant tokens, for example, by clicking them.

- The user can select a series of tokens by placing the cursor in an initial position (which can also be a token), holding down the SHIFT key, and clicking or tapping a new position. The tokens between these two cursor positions are then selected.

- In Desktop, hovering with the mouse over the token produces a tooltip with the entire content of the token.
